app/femr/data/models/mysql/PatientEncounter.java

```diff
@@ -56,7 +56,7 @@
     @JoinColumn(name = "patient_age_classification_id")
     private PatientAgeClassification patientAgeClassification;
     @ManyToOne(fetch = FetchType.EAGER)
-    @JoinColumn(name = "mission_trip_id")
+    @JoinColumn(name = "mission_city_id")
```

Comment:
The patient encounter column for trip is mission_trip_id, this should be reverted back

`@JoinColumn(name = "mission_trip_id")`